NEXT_CHANGES.rst

```diff
@@ -49,3 +49,14 @@ The workings of ``MSMeasurement.grab_flux`` are changed if ``plugins.USE_QUANT``
 In that case, it invokes the *quantifier* via ``MSMeasurement.grab_fluxes`` and returns
 just the flux of the requested molecule.
 
+New Zilien reader
+=================
+
+A new Zilien reader for the SpectroInlets' new Zilien dataset file format
+version. The new dataset version is able to integrate the Biologic EC-lab
+dataset during a measurement. Such dataset contains a new series with header
+EC-lab with two meta columns experiment_number and technique_number that the
+new reader is using during the process of creating Ixdat series objects. The
+objects created from the Zilien dataset match exactly the objects created from
+the Biologic mpt files, like when read one by one.
```

Comment:
Usage is the same like it was before. There will be just more series, when the TSV file includes also Biologic data.

When you want to get only the Biologic part, you need to create a `Measurement` from the TSV file, then grab by the `Biologic time/s` series and according to that take all related series that have such `tseries`.

Maybe there is a better way in the Ixdat. At this point, it is not possible to read the file and get **only** the Biologic part. It will always include our Zilien data. I don't see a point of reading a Zilien dataset, when you want only the Biologic data. You can simply `read_set()` directory with the MPT files.